pkg/claims/endorsement_test.go

```diff
@@ -68,6 +68,25 @@ func TestNotAfterBeforeNotBeforeEndorsement(t *testing.T) {
 	}
 }
 
+func TestGenerateProvenanceLessEndorsement(t *testing.T) {
+	newNotBefore := time.Now().AddDate(0, 0, 1)
```

Comment:
Ideally the tests (and the functions under tests) would not rely on the current time, but would accept a reference time from the outside. This will make them easier to test (without having to depend on relative times), and also avoid actual bugs (TOCTOU and related issues, if multiple functions get their own `time.Now()` independently at slightly different times).

Reply:
For the endorsement to be valid the notBefore time should be after the `issued` time, which uses the current time. For the test to pass, we have to use the current time here.